### PR TITLE
Setup graphene

### PR DESCRIPTION
**Pull request by bluquar**
This allows us to expose our models from django via GraphQL.

Following this tutorial: https://docs.graphene-python.org/projects/django/en/latest/tutorial-plain/

Screen Shot 2021-11-20 at 4 09 20 AM

### CODE REVIEW

File: backend/community_aid_backend/admin.py [hunk 1]
```diff
@@ -1,3 +1,12 @@
 from django.contrib import admin
+from .models import AidRequest
 
-# Register your models here.
+class AidRequestAdmin(admin.ModelAdmin):
```
Comment (lowellbander) [inline backend/community_aid_backend/admin.py:4]: These changes are the same as in #15 right? Are these actually two separate branches with the same changes? idk how github works

File: backend/community_aid_backend/schema.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        model = AidRequest
+        fields = (
+            'what_is_needed', 
+            'completed', 
+            'zip_code',
+            'created_at',
+        )
+
+class Query(graphene.ObjectType):
+    all_aid_requests = graphene.List(AidRequestType)
```
Comment (lowellbander) [inline backend/community_aid_backend/schema.py:17]: I don't see this explicitly referenced, is it implicitly referenced by the resolve_ method below?

File: backend/community_aid_backend/models.py [hunk 3]
```diff
@@ -1,3 +1,12 @@
...
 # Create your models here.
+
+class AidRequest(models.Model):
+    what_is_needed = models.CharField(max_length=120)
+    completed = models.BooleanField(default=False)
+    zip_code = models.PositiveIntegerField()
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    def _str_(self):
+        return self.what_is_needed
```
Comment (lowellbander) [inline backend/community_aid_backend/models.py:12]: is this just for the admin view? probs fine, but could also do like

`return f'{self.what_is_needed} ({'complete' if self.completed else 'incomplete'}, {self.zip_code})'`

to include some more contet

**Review by lowellbander — APPROVED** [review]
whoa! graphiql in the wild! real good work bud!

File: backend/community_aid_backend/admin.py (hunk 1 above)
Comment (bluquar) [reply backend/community_aid_backend/admin.py:4]: yeah, let me see if I can fix this. this branch contains some of the same commits as that branch

File: backend/community_aid_backend/schema.py (hunk 2 above)
Comment (bluquar) [reply backend/community_aid_backend/schema.py:17]: I believe it's automatically referenced by graphene by virtue of being a property of this class

File: backend/community_aid_backend/models.py (hunk 3 above)
Comment (bluquar) [reply backend/community_aid_backend/models.py:12]: honestly I have no idea what this is for lol I just copied it from the documentation

File: backend/community_aid_backend/admin.py (hunk 1 above)
Comment (bluquar) [reply backend/community_aid_backend/admin.py:4]: wow I have no idea how to fix this. everything I try just adds more commits lol

**Comment by bluquar** [comment]
abandoning in favor of #34

**Pull request closed without merging**